views/applications/all.html

```diff
@@ -0,0 +1,49 @@
+{% extends "_donut.html" %}
+
+{% block pageHead %}
+  {{ super() }}
+{% endblock %}
+
+{% block header %}
+  {{ super() }}
+{% endblock %}
+
+{% block main %}
+  <div class="introduction">
+    <h1 class="introduction--h1">{{structure.name}}</h1>
+    <p class="introduction--p">{{structure.description}}</p>
+    <a href="/${config.base}/${config.actions.add}" class="introduction--button">{{config.actions.add|capitalize}} {{config.base}}</a>
+  </div>
+
+  {% if applications %}
+  <table class="base--table">
+    <thead class="base--thead">
+      <tr class="base--tr">
+        <th class="base--th">Application</th>
+        <th class="base--th">Created</th>
+        <th class="base--th">Live</th>
+        <th class="base--th">Updated</th>
+        <th class="base--th">Sunset</th>
+        <th class="base--th">Edit</th>
+      </tr>
+    </thead>
+    <tbody class="base--tbody">
+    {% for a in applications %}
+      <tr class="base--tr">
+        <td data-th="Application" class="base--td">{{a.name}}</td>
+        <td data-th="Created" class="base--td">{{a.created}}</td>
+        <td data-th="Live" class="base--td">{{a['live-endpoint']}}</td>
```

Comment:
`{% if a.endpoints.live %} {{a.endpoints.live}} {% endif %}` where `a.endpoints.live` is either `'pass'`, `'fail'` based on the results of the webhook (from the audit trail). Wrap it in a span with a `pass/fail` class so we have styling hooks later on. That's the entire extent of this addition. Or, if you'd like, `✓` for pass and `✗` for fail